dev_guide/projects.adoc

```diff
@@ -165,3 +165,36 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+OpenShift cluster administrators can set the cluster-wide default node selector to restrict pod placement to specific nodes. As an OpenShift developer, you can set a node selector on a pod configuration to restrict nodes even further.
```

Comment:
"OpenShift cluster administrators can set the cluster-wide default node selector" well, it could be cluster-wide or specific to your project. Since we don't give them any way to know the difference, might as well just say "OpenShift cluster administrators can set the default node selector for your project ..."